compiler/compile/OMRCompilation.hpp

```diff
@@ -1052,6 +1052,10 @@ class OMR_EXTENSIBLE Compilation
     */
    DebugCounterMap &getDebugCounterMap() { return _debugCounterMap; }
 
+   // J9
+   // For JITServer: returns true if the compilation takes place at the server.
```

Comment:
I would rather you turn this into a more generic query about whether this compile occurs out-of-process rather than tying it to "JITServer" and "J9".

If this really is a J9-specific query with J9-specific semantics then I question why it is implemented here and not in the J9-specific version of this class.

Reply:
The idea was to make this query quite generic as there are ideas of implementing out-of-process compilation outside openj9. As such, I would just remove the comment referring to openj9.
Openj9 already has its own implementation of `isOutOfProcessCompilation()`

Reply:
That's fine.  But please replace the removed comment with a proper Doxygen comment as we usually require for new APIs.

Reply:
Will add Doxygen comment for the new API.